packages/suite/src/views/wallet/send/Options/StellarOptions/StellarMemo.tsx

```diff
@@ -0,0 +1,44 @@
+import { Input, Icon } from '@trezor/components';
+import { getInputState } from '@suite-common/wallet-utils';
+import { formInputsMaxLength } from '@suite-common/validators';
+
+import { Translation } from 'src/components/suite';
+import { useSendFormContext } from 'src/hooks/wallet';
+import { useTranslation } from 'src/hooks/suite';
+
+interface StellarMemoProps {
+    close: () => void;
+}
+
+export const StellarMemo = ({ close }: StellarMemoProps) => {
```

Comment:
Please check if `DestinationTag` component can be made generic

Reply:
https://github.com/trezor/trezor-suite/pull/16483/files#diff-fd866cc0695afb3249f717224024fc84bcb5b462926eab976666d8acf0a514cbR23